templates/ehub.json

```diff
@@ -3,52 +3,52 @@
     "contentVersion": "1.0.0.1",
```

Comment:
just one small thing: could you please bump `contentVersion`? It has no functional effect but it can be helpful during troubleshooting to differentiate between template versions

Reply:
resolved thank you